### DISCUSSION TITLE
Support for new 340 Series Cameras?

### DISCUSSION THREAD

**Discussion by same2you** [fuatakgun/eufy_security#949] [General]
Wondering if there are plans to add support for the S340 Solar Camera/ E340 Floodlight Cam/ E340 Doorbell any time soon?  I'm only finding the debug and trigger/reset alarm entities.

**Comment by fuatakgun** [comment]
Check with bropat through addon, integration would support it automatically

**Comment by Golgautier** [comment]
@same2you, have you reached out Bropat ?

**Comment by Golgautier** [comment]
Yes you did : https://github.com/bropat/eufy-security-ws/issues/275
